storage/kubernetes.go

```diff
@@ -57,3 +57,16 @@ type KubernetesFilesystemInfo struct {
 	// Size is the size of the filesystem in MiB.
 	Size uint64
 }
+
+type KubernetesVolumeParams struct {
+	StorageName string
+	Size        uint64
+	Provider    ProviderType
+	Attributes  map[string]interface{}
+	Tags        map[string]string
```

Comment:
Several of these aren't currently used.
For Tags, we should call them ResourceTags to be consistent.
We should use them to set the annotations on the PVC